### PR TITLE
Sensor

### PR DESCRIPTION
**Pull request by yongan007**
This is my sensor homework.

### CODE REVIEW

File: src/asimo_control/script/camera.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    except CvBridgeError, e:
+        print(e)
+    else:
+        # Save your OpenCV2 image as a jpeg 
+        # if t < 0:
+        t = str(int(time.time())-t0+1)
+        #print(t)
+        # else:
+        if int(t) <= 10:
+            cv2.imwrite('/home/yong_ros/example_ws/src/asimo_e1_description/Photos/camera_image'+t+'.jpeg', cv2_img)
```
Comment (l1va) [inline src/asimo_control/script/camera.py:40]: hardcode that will not work on my laptop

File: src/asimo_e1_description/launch/my_world.world [hunk 2]
```diff
@@ -0,0 +1,1850 @@
...
+              <uri>file://media/materials/scripts/gazebo.material</uri>
+            </script>
+          </material>
+        </visual>
+        <visual name='body_fixed_joint_lump__hokuyo_visual_2'>
+          <pose frame=''>0.5 0 3.9 0 -0 0</pose>
+          <geometry>
+            <mesh>
+              <scale>1 1 1</scale>
+              <uri>/home/yong_ros/example_ws/src/asimo_e1_description/meshes/hokuyo.dae</uri>
```
Comment (l1va) [inline src/asimo_e1_description/launch/my_world.world:199]: bad to use full path

File: src/asimo_e1_description/launch/my_world.world [hunk 3]
```diff
@@ -0,0 +1,1850 @@
...
+    </audio>
+    <wind/>
+    <spherical_coordinates>
+      <surface_model>EARTH_WGS84</surface_model>
+      <latitude_deg>0</latitude_deg>
+      <longitude_deg>0</longitude_deg>
+      <elevation>0</elevation>
+      <heading_deg>0</heading_deg>
+    </spherical_coordinates>
+    <model name='myoboat'>
```
Comment (l1va) [inline src/asimo_e1_description/launch/my_world.world:88]: please use robot from urdf (or xacro). Leave world file only for other static subjects.

**Review by l1va — CHANGES_REQUESTED** [review]

File: src/asimo_control/script/camera.py (hunk 1 above)
Comment (yongan007) [reply src/asimo_control/script/camera.py:40]: Full path is removed.

File: src/asimo_e1_description/launch/my_world.world (hunk 2 above)
Comment (yongan007) [reply src/asimo_e1_description/launch/my_world.world:199]: Completed your suggestion.

File: src/asimo_e1_description/launch/my_world.world (hunk 3 above)
Comment (yongan007) [reply src/asimo_e1_description/launch/my_world.world:88]: Completed your suggestion.

File: src/asimo_e1_description/launch/my_world.world (hunk 2 above)
Comment (l1va) [reply src/asimo_e1_description/launch/my_world.world:199]: do not see

**Pull request merged**